wow/core/src/main/scala/wow/auth/data/Account.scala

```diff
@@ -39,18 +39,14 @@ object Account extends SQLSyntaxSupport[Account] with RichColumn[Account] {
     Account(id, login, identity, sessionKey)
   }
 
-  def find(condition: SQLSyntax)(implicit session: DBSession = autoSession): Option[Account] = withSQL {
-    select(column.*)
-      .from(Account as syntax)
-      .where
-      .append(condition)
-  }.map(Account(syntax)).single.apply()
-
-  def findByLogin(login: String)(implicit session: DBSession = autoSession): Option[Account] =
-    find(sqls.eq(c.login, login))
-
-  def findById(id: Long)(implicit session: DBSession = autoSession): Option[Account] =
-    find(sqls.eq(c.id, id))
+  def create(login: String, identity: Srp6Identity)(implicit session: DBSession = autoSession): Int = withSQL {
+    insert.into(Account)
+      .namedValues(
+        c.login -> login.toUpperCase(),
+        c.salt -> identity.salt,
+        c.verifier -> identity.verifier
+      )
+  }.updateAndReturnGeneratedKey.apply().toInt
 
   def save(account: Account)(implicit session: DBSession = autoSession): Int = withSQL {
```

Comment:
For the naming conventions, I followed those I found on the [scalikejdbc website](http://scalikejdbc.org/documentation/reverse-engineering.html).
Since the objects we are manipulating are Accounts and not Sessions, it would feel better to use the word "account", but then what would be the point of having something like "Account.saveAccount(x)" ? I feel like this would be redundant.

Note that if other fields are added to the Account object, then those methods should be changed such that they also update those fields (i.e. those methods are not dedicated to modifying the "session" aspect of an Account, but all the fields in the object).